frontend/src/components/LogDetail/index.tsx

```diff
@@ -126,8 +146,8 @@ function LogDetail({
 		>
 			<div className="log-detail-drawer__log">
 				<Divider type="vertical" className={cx('log-type-indicator', logType)} />
-				<Tooltip title={log?.body} placement="left">
-					<Typography.Text className="log-body">{log?.body}</Typography.Text>
+				<Tooltip title={removeEscapeCharacters(log?.body)} placement="left">
+					<div className="log-body" dangerouslySetInnerHTML={htmlBody} />
```

Comment:
@vikrantgupta25 : I remember someone had pointed out the risk of this and we had replied that it must be safe. Just look into that thread to confirm we are not reverting the change.

Reply:
i remember the conversation, we are using dompurify everywhere so no such risk with this. already using this in logs raw and other views